src/transformer.d.ts

```diff
@@ -4,7 +4,7 @@ import type {DefaultTheme} from "tailwindcss/types/generated/default-theme";
 export type DefaultScreens = keyof DefaultTheme["screens"];
 
 export type WithTV = {
-  <C extends Config = {}>(tvConfig: Config): C;
+  (tvConfig: Config): Config;
```

Comment:
IMO, it would be better to make `tvConfig` use the generic type `C`, so that the returned type can be inferred based on the input configs when people need to further expand the configs.

```typescript
export type WithTV = {
  <C extends Config>(tvConfig: C): C;
};
```

Reply:
Updated